_docs/_latest/develop/test/integration-testing/run-test-suite.md

```diff
@@ -0,0 +1,148 @@
+---
+title: Run The Test Suite Locally
+order: 045
+---
+
+# {{ page.title }}
+
+This page describes how to run the integration test suite for your Forseti
+contributions on your local dev environment. 
+
+## **Setting up development environment**
+
+- Create a new project in your organization.
+- Terraform uses an IAM Service Account to deploy and configure resources on 
+behalf of the user. You can create the Service account, and grant the necessary 
+roles to the Service Account manually or by running a helper script. 
+
+### **Creating Service Account by running the helper script**
+
+In order to execute this script, you must have an account with the following 
+list of permissions: 
+* resourcemanager.organizations.list
```

Comment:
Since the roles are listed below, can we just list the roles and not the permissions? I think it's easiest if we just tell people to generate an SA with the helper script. We already have documentation on the website that covers the roles necessary to deploy Forseti.

Reply:
Have removed the permissions that were listed. I have documented both ways to generate the Service Account so users can decide just like we have documented both ways to create the Service Account to deploy Forseti. 

Running the helper script in this case would required additional roles to be granted. So, I have documented all the roles needed here to make the website user-friendly.